frontend/src/metabase/components/SandboxedPortal.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+function stop(event) {
+  event.stopPropagation();
+}
+
+// Prevent DOM events from bubbling through the React component tree
+// See https://reactjs.org/docs/portals.html#event-bubbling-through-portals
+function SandboxedPortal({ children, container }) {
+  return ReactDOM.createPortal(
+    <div
+      onClick={stop}
+      onContextMenu={stop}
+      onDoubleClick={stop}
+      onDrag={stop}
+      onDragEnd={stop}
+      onDragEnter={stop}
+      onDragExit={stop}
+      onDragLeave={stop}
+      onDragOver={stop}
+      onDragStart={stop}
+      onDrop={stop}
+      onMouseDown={stop}
+      onMouseEnter={stop}
+      onMouseLeave={stop}
+      onMouseMove={stop}
+      onMouseOver={stop}
+      onMouseOut={stop}
+      onMouseUp={stop}
+      onKeyDown={stop}
+      onKeyPress={stop}
+      onKeyUp={stop}
+      onFocus={stop}
+      onBlur={stop}
+      onChange={stop}
+      onInput={stop}
+      onInvalid={stop}
+      onSubmit={stop}
```

Comment:
Will this list stay relatively stable?

Reply:
I think so. This is specifically to deal with our implementation of `Triggerable` which wraps `Popover` in a clickable anchor tag. In general, I think we should avoid this pattern going forward, so hopefully we won't need `SandboxedPortal` one day. Now that I'm looking at `Triggerable` again I realize that we might be able to just use a fragment to make the portal child a sibling of the anchor tag. I'll look into that soon.